### ISSUE TITLE
minor test failure

### ISSUE THREAD

**Issue by bastistician** [config-i1/smooth#205]
Checking the current CRAN version 3.2.1, this test

https://github.com/config-i1/smooth/blob/d65e2f7d5430717185ce561cac275bb46c30a61b/tests/testthat/test_es.R#L10

fails on Alpine Linux, possibly due to a lack of numerical tolerance:

```
  ══ Failed tests ════════════════════════════════════════════════════════════════
  ── Failure ('test_es.R:10:5'): Test damped-trend ETS on BJsales ────────────────
  round(es(BJsales, model = "AAdN", silent = TRUE)$phi, 2) not equal to 0.86.
  1/1 mismatches
  [1] 0.85 - 0.86 == -0.01
  
  [ FAIL 1 | WARN 0 | SKIP 74 | PASS 33 ]
```

**Comment by config-i1** [comment]
Thanks! Yes, this is because of the tolerance. I'll fix it.

**Comment by config-i1** [comment]
Should be fixed now in 468949b1f4f4a72cc638c352242a3c4ae4d07375
